[2024-12-06T20:34:33.943+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T20:34:34.027+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: homework_dag.bigquery_external_table scheduled__2021-04-05T06:00:00+00:00 [queued]>
[2024-12-06T20:34:34.054+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: homework_dag.bigquery_external_table scheduled__2021-04-05T06:00:00+00:00 [queued]>
[2024-12-06T20:34:34.060+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-06T20:34:34.099+0000] {taskinstance.py:2889} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table> on 2021-04-05 06:00:00+00:00
[2024-12-06T20:34:34.117+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4472) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T20:34:34.120+0000] {standard_task_runner.py:72} INFO - Started process 4476 to run task
[2024-12-06T20:34:34.120+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'homework_dag', 'bigquery_external_table', 'scheduled__2021-04-05T06:00:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/homework_dag.py', '--cfg-path', '/tmp/tmp5bna4muo']
[2024-12-06T20:34:34.125+0000] {standard_task_runner.py:105} INFO - Job 130: Subtask bigquery_external_table
[2024-12-06T20:34:34.194+0000] {task_command.py:467} INFO - Running <TaskInstance: homework_dag.bigquery_external_table scheduled__2021-04-05T06:00:00+00:00 [running]> on host 39ecaeffba2c
[2024-12-06T20:34:34.242+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='homework_dag' AIRFLOW_CTX_TASK_ID='bigquery_external_table' AIRFLOW_CTX_EXECUTION_DATE='2021-04-05T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-05T06:00:00+00:00'
[2024-12-06T20:34:34.243+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T20:34:34.252+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-12-06T20:34:34.252+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-12-06T20:34:34.253+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2024-12-06T20:34:35.001+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T20:34:35.001+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=homework_dag, task_id=bigquery_external_table, run_id=scheduled__2021-04-05T06:00:00+00:00, execution_date=20210405T060000, start_date=20241206T203434, end_date=20241206T203435
[2024-12-06T20:34:35.015+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T20:34:35.028+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-06T20:34:35.030+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
